micropython/modules_py/interstate75.md

```diff
@@ -44,7 +44,7 @@ import interstate75
 
 display = interstate75.DISPLAY_INTERSTATE75_32X32
 
-board = interstate75.Interstate75(display=display, interstate75w=True)
+board = interstate75.Interstate75(display=display, interstate75w=False)  #This is th e setup to be able to access the both A and BOOT on the Interstate75 (non W)
```

Comment:
"th e" should be "the"

Reply:
Oh and a space is needed after the #, otherwise linting will shout

Reply:
Opps slip of the keyboard :)